src/test/scenarios/datafactory/output/src/datafactory/azext_datafactory/generated/_help.py

```diff
@@ -23,7 +23,7 @@
     examples:
       - name: Factories_ListByResourceGroup
         text: |-
-               az datafactory list --resource-group "exampleResourceGroup"
+               az datafactory list --resource-group "myResourceGroup2"
```

Comment:
Why this is myResourceGroup2

Reply:
this should be the second resource-group in all example files

Reply:
Why? They are exampleResourceGroup in all the examples.

Reply:
fixed here: https://github.com/Azure/autorest.az/pull/462/commits/6ec8b8a729c490e174ee53afdb633351604a1c0f